src/main/resources/standardTestCases/curl.test

```diff
@@ -0,0 +1,11 @@
+name: curl
```

Comment:
Since this test isn't looking at output from the page, would you agree that #21 could be closed?